src/applications/check-in/components/PreCheckInSuccessAlert.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { useSendPreCheckInData } from '../hooks/useSendPreCheckInData';
+
+import { makeSelectVeteranData } from '../selectors';
+import { hasPhoneAppointments } from '../utils/appointment';
+
+const PreCheckInSuccessAlert = () => {
+  const selectVeteranData = useMemo(makeSelectVeteranData, []);
+  const { appointments } = useSelector(selectVeteranData);
+  const { t } = useTranslation();
+
+  const { isLoading } = useSendPreCheckInData();
+
+  const successMessage = hasPhoneAppointments(appointments)
+    ? t('your-provider-will-call-you-at-your-appointment-time')
+    : t('you-can-check-in-with-your-smartphone', {
+        date: new Date(appointments[0].startTime),
+      });
+
+  if (isLoading) {
+    return (
+      <va-loading-indicator
+        data-testid="loading-indicator"
+        message={t('setting-pre-check-in-completed')}
+      />
+    );
+  }
+
+  return (
+    <section data-testid="pre-check-in-success-alert">
+      <va-alert
+        close-btn-aria-label="Close notification"
+        closeable
```

Comment:
not related to this ticket, but I noticed that these alerts don't close. I wonder if we need to pass a close func to it?